package.json

```diff
@@ -283,7 +283,12 @@
       },
       {
         "command": "one.editor.mpq.createFromDefaultExplorer",
-        "title": "Create MPQ json",
+        "title": "ONE: Create Mixed Precision Quantization Configuration",
```

Comment:
Do we need `ONE:` prefix in title?

Reply:
It's the menu shown from the default explorer, so I added it. 
The menu on ONE Explorer doesn't include "ONE:" instead.

Reply:
Actually, I am rather worried that ONE Explorer cannot show the created mpq configuration files for now. 😨